extensions/standard-processors/processors/GetTCP.cpp

```diff
@@ -148,7 +149,13 @@ void GetTCP::onSchedule(const std::shared_ptr<core::ProcessContext> &context, co
 
   logger_->log_trace("EOM is defined as %i", endOfMessageByte);
 
-  context->getProperty(ReconnectInterval.getName(), reconnect_interval_);
+  std::string reconnect_interval_str;
+  if (context->getProperty(ReconnectInterval.getName(), reconnect_interval_str) &&
+      core::Property::getTimeMSFromString(reconnect_interval_str, reconnect_interval_)) {
+    logger_->log_debug("Reconnect interval is %d ms", reconnect_interval_);
```

Comment:
Minor but this might overflow for `uint64_t`:
https://godbolt.org/z/WsTK8W

Reply:
Done in [c0110dc](https://github.com/apache/nifi-minifi-cpp/pull/959/commits/c0110dca0937ac4af93e32d7ccab8347132183c7)